scout/server/blueprints/variant/templates/variant/variant.html

```diff
@@ -586,27 +586,38 @@ <h4 class="modal-title">Cancel Sanger sequencing</h4>
           <a href="{{ gene.reactome_link }}" class="btn btn-outline-secondary" target="_blank">Reactome</a>
           <a href="{{ gene.expression_atlas_link }}" class="btn btn-outline-secondary" target="_blank">Expr. Atlas</a>
           <a href="{{ gene.clingen_link }}" class="btn btn-outline-secondary" target="_blank">ClinGen</a>
+          <!-- Display Alamut link only if bam is available -->
           {% if case.mt_bams or case.bam_files %}
-            <a href="{{ variant.alamut_link }}" class="btn btn-outline-secondary" target="_blank">Alamut</a>
+            {% set alamut_position = [ "GRCh38:" if case.genome_build == 38 ,  variant.chromosome, ":", variant.position, variant.reference, ">", variant.alternative]  %}
+            <form>
+            {% if variant.chromosome == "MT" and case.mt_bams %}
+              <input type="button" class="btn btn-outline-secondary" value='Alamut' onclick='httpGetLoadBam({{ alamut_position|join() }} ,
+                {{url_for("alignviewers.unindexed_remote_static", file=case.mt_bams[0])}} );' />
+            {% elif case.bam_files %} <!-- not a MT variant and bam file available -->
+            <input type="button" class="btn btn-outline-secondary" value='Alamut' onclick='httpGetLoadBam({{ alamut_position|join() }} ,
+              {{url_for("alignviewers.unindexed_remote_static", file=case.mt_bams[0])}} );' />
```

Comment:
Oops; not the mt_bam here, right?
Also probably not the unindexed url for the whole file.. I wonder how this would work with auth? Does it pass the token in the url?

Reply:
> Does it pass the token in the url?
Not that I'm aware of.

The problem with using the other method is that it will pass just a part of the file, see the code:
![Screenshot from 2020-02-26 08-33-56](https://user-images.githubusercontent.com/28093618/75322378-e5044600-5872-11ea-87e1-f5a34653d0e0.png)

I'm not sure alamut likes it.

I've fixed the path to the right bam for the non-MT link now

Reply:
We'll have to test I suppose: I'm not so sure Alamut will be able to reach it without a token, but lets see. But sure, lets start with the full file for testing. That won't work for whole genomes though, so if Alamut can't do ranged/some sort of streamed queries, we will likely have to check for the sequencing type before adding that to the url. Trying to dl a full bam via the hospital network means waiting for a few h and likely failing.. 😊

Reply:
If it won' work with whole genome then there is no much point. We could try directly with passing ranges. I'll fix it as soon as I finish something else I'm doing

Reply:
Well, I guess the question came from the cancer side, where we have the tiny panels. I guess those could be dl:ed in full if need be, much like the mito dito..